isis/scripts/isisStartup.csh

```diff
@@ -0,0 +1,37 @@
+#!/bin/csh
+################################################################################
+# This file should be sourced within your current shell using the "source"
+# command.
+#
+# On the command line type:
+# > setenv ISISROOT ????
+# > source isis3Startup.csh
+#
+# Replace the "????" in the above command line with the path you installed
+# the Isis distribution
+#
+################################################################################
+
+if ($?ISISROOT == 0) then
+  setenv ISISROOT /usgs/pkgs/isis3/install
```

Comment:
Do we still use this area? Should it use an anaconda package instead?

Reply:
Yeah, I would just return an error message/usage.